src/parse_policies.c

```diff
@@ -82,31 +82,31 @@ int count_char_in_str(const char * string, int len, char lookup) {
 
 // parse compaction policies in the following format: "max:1m;min:10s;avg:2h;avg:3d"
 // the format is AGGREGATION_FUNCTION:\d[s|m|h|d];
-int ParseCompactionPolicy(const char * policy_string, SimpleCompactionRule **parsed_rules_out,
-            size_t *rules_count) {
+int ParseCompactionPolicy(const char *policy_string, SimpleCompactionRule **parsed_rules_out,
+            uint64_t *rules_count) {
     char *token;
     char *token_iter_ptr;
-    size_t len = strlen(policy_string);
+    uint64_t len = strlen(policy_string);
```

Comment:
@ashtul size_t is a standard type to measure sizes, any reason why we should use the uint64_t instead?

Reply:
It is used in function count_char_in_str where we had and 'int'.